components/TripList/index.js

```diff
@@ -1,5 +1,6 @@
 import Trip from "../Trip";
 import useSWR from "swr";
+import { StyledUnorderedList } from "./TripList.styled";
 
 const fetcher = (...args) => fetch(...args).then((res) => res.json());
```

Comment:
Configure it globally in `_app.js` via `<SWRConfig>` ([see docs](https://swr.vercel.app/docs/global-configuration))

Reply:
We are not sure we understand what you mean by that, could you please explain further? Why should we configure our UL globally when we only use it once in one page?

Reply:
You are using `useSWR` to fetch your data on your homepage (trip list) and your details page. In both files you currently have defined a `fetcher`. Instead you can define the `fetcher` once in your `_app.js` and pass it to all your pages with the `<SWRConfig>`.

Reply:
Ahh, okay! :D Thank you for the clarification! This is going to be changed in refactoring, as this US is strictly about styling! 🎨